src/wallet/diagnose.h

```diff
@@ -736,10 +741,12 @@ class CheckDifficulty : public Diagnose
     }
 };
 /**
- * check ETTS
- * This is only checked if wallet is a researcher wallet because the purpose is to
+ * Dianose class to check ETTS.
```

Comment:
should be "Diagnose"

Reply:
Thank you for fixing the code. 
The code is actually much better and readable now.